account_invoice_validate_send_email/models/account_move.py

```diff
@@ -71,6 +71,10 @@ def action_post(self):
                 )
             ):
                 continue
+            if move.message_follower_ids.mapped("partner_id").filtered(
+                lambda x: (not x.user_ids or x.user_ids.share) and not x.email
+            ):
```

Comment:
```suggestion
            if not move.message_follower_ids.mapped("partner_id").filtered(
                lambda x: (not x.user_ids or x.user_ids.share) and x.email
            ):
```
Shouldn't it be like this?